src/pages/heejinkim/main/main.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'; 
+import Feed from '../Components/Feed';
+import Input from '../Components/Input'
+import "../main/main.scss";
+import "../../../styles/common.scss";
+import "../../../styles/reset.scss";
+import uuid from 'react-uuid'
+
+class Main extends React.Component {
+    constructor(){
+        super();
+        this.state = {
+            id: uuid(),
+            comment: '',
+            commentArr: [],
+            delComment: false,
+            commentData: []
+        }
+    }
+
+    handleInput = (e) => {
+        
+        this.setState({
+            comment : e.target.value
+            
+        })
+    }
+
+    handleSubmit = (e) => {
+         
+        e.preventDefault();
+        const newComment = {
+            mapId: this.state.id,
+            content: this.state.comment
+        }
+        console.log("newComment 잘 되나?", newComment)
+        const newCommentArr = [...this.state.commentArr, newComment]
+        
+        this.setState({
+            id: uuid(),
+            comment:'',
+            commentArr: newCommentArr,
+            delComment: false
+        })
+    }
+
+    componentDidMount() {
+        fetch('comment data 로컬 API 주소', {
+            method: "GET"
+        })
```

Comment:
get method는 default값으로 아래와 같이 생략가능합니다.
```suggestion
        fetch('comment data 로컬 API 주소')
```